### DISCUSSION TITLE
Install command not showing on artisan list

### DISCUSSION THREAD

**Discussion by jesperbjoernandersen** [spatie/laravel-package-tools#102] [Q&A] [answered]
Does anyone else have the issue when using $package->hasInstallCommand, where the package-name:install command doesn't appear on the list of commands when you type php artisan?

**Comment by freekmurze** [comment, marked as answer]
This is by design: I wanted to avoid that when you have 10 packages installed with an install command that the artisan command list would be flooded with install commands. 

Usually you only want to run the command once, when installing the package, so I thought it isn't really worthwhile to display it in the full list.

**Reply by jesperbjoernandersen** [reply]
Alright thank you. I was just worried that I was doing something wrong, since other packages, like volt, livewire etc. has the command showing.

Cheers :)
